lib/Applify.pm

```diff
@@ -270,11 +275,40 @@ sub _exit {
   exit $reason;
 }
 
+sub _generate_attribute_accessor {
+  my ($self, $args) = @_;
+  my $default = ref $args->{default} eq 'CODE' ? $args->{default} : sub { $args->{default} };
+  my $isa     = $args->{isa};
+  my $name    = $args->{name};
+
+  if (blessed $isa and $isa->can('check')) {
```

Comment:
Would it be better here to do an isa('Type::Tiny') check?

Reply:
This is on purpose, since I think there are other libraries than Type::Tiny that can be used.